app/scripts/modules/amazon/src/loadBalancer/details/targetGroupDetails.html

```diff
@@ -22,42 +22,21 @@ <h3 class="horizontal middle space-between flex-1" select-on-dbl-click>
         {{ctrl.targetGroup.name}}
       </h3>
     </div>
-    <div>
-      <div class="actions">
-        <div class="dropdown" uib-dropdown dropdown-append-to-body>
-          <button
-            type="button"
-            class="btn btn-sm btn-primary dropdown-toggle"
-            style="visibility: hidden"
-            disabled
-            uib-dropdown-toggle
-          >
-            Target Group Actions <span class="caret"></span>
-          </button>
-          <ul class="dropdown-menu" uib-dropdown-menu role="menu">
-            <li><a href ng-click="ctrl.editLoadBalancer()">Edit Target Group</a></li>
-            <li ng-if="!ctrl.loadBalancer.instances.length">
-              <a href ng-click="ctrl.deleteLoadBalancer()">Delete Target Group</a>
-            </li>
-            <li
-              ng-if="ctrl.loadBalancer.instances.length"
-              class="disabled"
-              uib-tooltip="You must detach all instances before you can delete this load balancer."
-            >
-              <a href ng-click="ctrl.deleteLoadBalancer()">Delete Target Group</a>
-            </li>
-          </ul>
-        </div>
-      </div>
-    </div>
```

Comment:
was this on purpose?

Reply:
oh yeah, forgot I did this otherwise would've commented it. all this markup is all hidden and seemingly has been forever — we don't actually let you mutate target groups directly from the details pane. Effectively the only noticeable change is that this silly whitespace goes away:

<img width="343" alt="Screen Shot 2019-06-27 at 9 34 19 PM" src="https://user-images.githubusercontent.com/1850998/60317350-81acc780-9923-11e9-8345-1206432b21f2.png">